### PR TITLE
Add Learn how it works link to cloud account

### LINKED ISSUE #13688: Home Assistant Cloud - documentation
### Feedback

It would be nice to have on the Home Assistant cloud page `http://<ha>/config/cloud/account` on every card a link to the according documentation topic. There is already a link "learn how it works" for the "remote control card" but it is missing for google `https://www.nabucasa.com/config/google_assistant/` and alexa card `https://www.nabucasa.com/config/amazon_alexa/`.

### URL

https://www.home-assistant.io/integrations/google_assistant/

### Version

2022.9.1

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Add "Learn how it works" link for Google Assistant and Alexa to be consistent with Remote UI card.
The sync button has been moved into the card. I'm not sure about the info text for sync action. Maybe we can explain when this action should be used?

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #13688 
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by matthiasdebaat** [comment]
Nice quick fix

**Review by bramkragten — DISMISSED** [review]

File: src/panels/config/cloud/account/cloud-alexa-pref.ts [hunk 1]
```diff
@@ -99,21 +99,42 @@ export class CloudAlexaPref extends LitElement {
...
+                        "ui.panel.config.cloud.account.alexa.button_sync_entities"
+                      )}
+                    </mwc-button>
+                  </div>
+                </div>
+                <p>
+                  ${this.hass!.localize(
+                    "ui.panel.config.cloud.account.alexa.info_sync_entities"
+                  )}
+                </p>
```
Comment (bramkragten) [inline src/panels/config/cloud/account/cloud-alexa-pref.ts:123]: Use a `ha-settings-row` here

Comment (piitaya) [reply src/panels/config/cloud/account/cloud-alexa-pref.ts]: image

Comment (piitaya) [reply src/panels/config/cloud/account/cloud-alexa-pref.ts]: More consistent with other panels

**Comment by balloob** [comment]
Syncing should basically happen automatically, it should not be put out so prominently. Maybe it can move into an overflow?

**Review by bramkragten — DISMISSED** [review]

**Comment by piitaya** [comment]
> Syncing should basically happen automatically, it should not be put out so prominently. Maybe it can move into an overflow? State reporting could move there too.

If we move it to the overflow, basically Alexa card will be empty 😅

**Comment by balloob** [comment]
I think that is okay. Just manage entities.

**Comment by piitaya** [comment]
We have to convert the switch to a button to enable Google or Alexa. And add the disable button into the overflow menu.
We also have to change the remote UI card to be consistent. Do you think it's worth it?

**Comment by balloob** [comment]
We could put the overflow menu in the bottom bar and keep the switch.

**Comment by piitaya** [comment]
I didn't known material guidelines allowed it 😅

image

**Comment by piitaya** [comment]
I will look with @matthiasdebaat how we can improve the card because there is no space left on mobile

**Comment by piitaya** [comment]
- [x] Help link is moved in the card header
- [x] Report state is removed
- [x] Sync entities is moved to detail page (an overflow menu has been added)

# Google Assistant / Alexa

Capture d’écran 2022-09-13 à 18 00 55

# Remote

Capture d’écran 2022-09-13 à 18 01 02

# Google Assistant / Alexa page

Capture d’écran 2022-09-13 à 18 56 28

**Comment by bramkragten** [comment]
Report state is remove completely now?

**Comment by piitaya** [comment]
Yes because it doesn't make sense to disable it. It's enable by default.
But we need a back end PR to ensure it's activated for all. Otherwise, we can restore state reporting in the card if we need 😅

**Comment by bramkragten** [comment]
> Yes because it doesn't make sense to disable it. It's enable by default. But we need a back end PR to ensure it's enabled for all. Otherwise, we can restore state reporting in the card if we need 😅

I think we should keep it, but in an overflow or something

**Comment by piitaya** [comment]
I restored the report state toggle so we can ship it before the beta 🙂 I will create another PR to remove it when the back-end PR to enable it for all users is done.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**